test/BF16/mlp-all-bf16.mlir

```diff
@@ -41,7 +41,9 @@ module @predict_function {
     linalg.fill ins(%c1:bf16) outs(%interim1: memref<128x512xbf16>)
     %threshold = arith.constant 0.75:bf16
     check.expect_almost_eq(%interim1, %arg12, %threshold): memref<128x512xbf16>, memref<128x512xbf16>, bf16
- 
+    memref.dealloc %interim1: memref<128x512xbf16>
```

Comment:
Do we need the two tests? If the kernels are the same, and the other is testing tpp-run as well, than this test can be removed.

Reply:
You're right, I'll remove this one. It's doing the same thing as tpprun test except initializations are explicit. We can reintroduce it after random initialization is implemented in tpp-run.